.github/workflows/build.yml

```diff
@@ -10,10 +15,12 @@ jobs:
 
     steps:
       - name: Checkout 🛎
-        uses: actions/checkout@v4
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+        with:
+          persist-credentials: false
```

Comment:
GitHub's `actions/checkout` abides by SemVer, so fixing this sortof-vulnerability isn't going to happen soon. I don't think it's exploitable here, but fixing it doesn't hurt.